kumascript/macros/DOMxRef.ejs

```diff
@@ -20,7 +20,8 @@ let api = $0;
 let str = $1 || $0;
 let rtlLocales = ['ar', 'he', 'fa'];
 
-api = api.replace(/ /g, '_')
+api = api.replace(/: /g, '/')
```

Comment:
Is that space there deliberate?
I think it might be overly surprising that it doesn't work if someone types: `{{domxref("ReadableStream:getReader()")}}`.

Reply:
I think it should be: `/:\s*/g`

Reply:
> Is that space there deliberate?

The space is deliberate, yes. See https://discourse.mozilla.org/t/incorrect-titles-for-method-property-articles/31641/17 for the context.



> I think it should be: `/:\s*/g`

That would be OK by me if we all have agreement about it.